crate_universe/src/rendering.rs

```diff
@@ -766,6 +768,7 @@ mod test {
 
         assert!(build_file_content.contains("rust_library("));
```

Comment:
Validate `crate-name` tag correctness for:
1. `rust_library`
2. `cargo_build_script`
3. `rust_proc_macro`
4. `rust_binary`